java8/Java8.g4

```diff
@@ -348,7 +348,8 @@ fieldModifier
 	;
 
 variableDeclaratorList
-	:	variableDeclarator (',' variableDeclarator)*
+	:	variableDeclarator
+	|	variableDeclaratorList ',' variableDeclarator
```

Comment:
I adhere the following priority:
1. Rule count.
2. If possible replace left-recursive rule with non-left-recursive or no-recursion rule (your case).